### PR TITLE
New version: ICBC.ICBCEBankAssist version 2.1.3.0

### PR DESCRIPTION
**Pull request by OfficialEsco**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

* Resolves #57972
* Resolves #57971

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/58002)

### CODE REVIEW

File: manifests/i/ICBC/ICBCEBankAssist/2.1.3.0/ICBC.ICBCEBankAssist.installer.yaml [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 InstallerType: wix
 Scope: machine
 UpgradeBehavior: install
 Installers:
 - Architecture: x64
   InstallerUrl: http://www.icbc.com.cn/ICBC/html/download/EbankToolsSoftware/x64/ICBCSetupIntegration_64.msi
-  InstallerSha256: 747504B953D45B95F65E6B3D397D3AF40FAE4E6890FE92274AB3CBFF76791907
-  ProductCode: '{076BD7BB-706A-4046-92CC-1DDF1710518C}'
+  InstallerSha256: 0100D62754AA2D04D7E1B713E1A8A2FECFCA53E341CA456BE99806359AA4955F
+  ProductCode: '{B2B1CE58-000C-4D53-BCD1-F3FE6FA9FCCB}'
```
Comment (OfficialEsco) [inline manifests/i/ICBC/ICBCEBankAssist/2.1.3.0/ICBC.ICBCEBankAssist.installer.yaml:14]: Downloading the old Installer

```
4/17/2022 9:37:15 AM
Status: Completed
##[error] Installer http://www.icbc.com.cn/ICBC/html/download/EbankToolsSoftware/x64/ICBCSetupIntegration_64.msi
##[error] Result: Failed
##vso[task.logissue type=error] Specified hash doesn't match.
Actual hash:   747504B953D45B95F65E6B3D397D3AF40FAE4E6890FE92274AB3CBFF76791907
Expected hash: 0100d62754aa2d04d7e1b713e1a8a2fecfca53e341ca456be99806359aa4955f
```

**Review by ItzLevvie — APPROVED** [review]

**Comment by lechacon** [comment]
> wingetbot waivers Add Validation-Domain

Ha, I clicked the wrong button 😆

**Pull request merged**